docs/reference/cli.md

```diff
@@ -181,34 +178,32 @@ Before watching a React package, it must be installed and linked. This can be do
 
 - `-l`/`--link` Install and link the JavaScript/React package before watching for changes.
 
-
 !!! info
-    The command ``assets watch-module`` only rebuilds the JavaScript/React package into InvenioRDM's Webpack project. However, before you can
-    see the changes, you must also rebuild InvenioRDM's Webpack project. Hence, you normally always use ``assets watch-module`` and ``assets watch`` in together:
+The command `assets watch-module` only rebuilds the JavaScript/React package into InvenioRDM's Webpack project. However, before you can
+see the changes, you must also rebuild InvenioRDM's Webpack project. Hence, you normally always use `assets watch-module` and `assets watch` in together:
```

Comment:
  ┬┴┤(･_├┬┴

![image](https://user-images.githubusercontent.com/1932651/133842784-4358556d-fd10-421d-a11f-b257ace9f2b0.png)

should be https://inveniordm.docs.cern.ch/reference/cli/#assets-watch-module

No worries we'll get there :)